services/streammanager/googlesheets/schemas/googlesheetsschema.go

```diff
@@ -0,0 +1,85 @@
+package schemas
+
+import (
+	"github.com/tidwall/gjson"
+)
+
+func GetSheetsHeaderSchema() []interface{} {
+	hschema := []interface{}{
+		"id",
```

Comment:
Can we reuse these fields instead of listing them again in the next function?

Reply:
We have addressed this in the new change